### ISSUE TITLE
Practice App: Implementing API for comment-info - Unit Tests

### ISSUE THREAD

**Issue by Keremgunduz7** [bounswe/bounswe2022group5#111]
### Description:

* The tests will be implemented for the comment-info API of the Practice-App. 
* TestCase library from django.test will be used.

**What to do:**
- [x] Write tests for urls.
- [x] Write tests for views.

**Reviewers:** @palahb @oguzhandemirelx   

**Task Deadline:** 16.05.2022 16:00
**Review Deadline:** 16.05.2022 20:00

**Comment by Keremgunduz7** [comment]
Unit tests for the comment-info endpoint are implemented.

4 URL tests and,
15 functional tests implemented.

Committed on a new branch named with this issue.

After a little bit of review, we can close this issue and merge the commit on the master. @palahb , @oguzhandemirelx

**Comment by Keremgunduz7** [comment]
A simple problem with the URL's emerged.
After pulling the master branch to my branch, occurred some conflicts with the urls.

We have used same names in some index functions with my group mates. 

I have solved the problem by appending a prefix of "comment-info" to my urls and to my functions in the /comment_info/views.py 

Since it was a simple error, I didn't open a new branch and corrected the issue in this branch.
